### ISSUE TITLE
lemma (still/again?) handled incorrectly

### ISSUE THREAD

**Issue by floriandk** [stenskjaer/samewords#23]
_As I suspect that you aren't notified of comments on closed issues and I cannot reopen them either, I put this here as a new issue even though I've already written this in #13 :_

Is the rewritten code you mention included in the vers. 0.2.7 branch "issue-19"? Testing _any_ ellipsis marker, with or without calling the json file give me an incorrect result, e.g. 

```latex
\documentclass{article}

\usepackage[series={A},nofamiliar,noeledsec,noledgroup,draft]{reledmac}

\begin{document}

\beginnumbering
\pstart
\edtext{B}{\Afootnote{del.}}
\edtext{F}{\Afootnote{del.}}
\edtext{B C D E F}{\lemma{B--F}\Afootnote{b c d e f}}
\edtext{B}{\Afootnote{bb}}
\edtext{F}{\Afootnote{ff}}
\pend
\endnumbering


\end{document}
```

 gives:

```latex
\documentclass{article}

\usepackage[series={A},nofamiliar,noeledsec,noledgroup,draft]{reledmac}

\begin{document}

\beginnumbering
\pstart
\edtext{\sameword[1]{B}}{\Afootnote{del.}}
\edtext{\sameword[1]{F}}{\Afootnote{del.}}
\edtext{\sameword{B} C D E \sameword{F}}{\lemma{B--F}\Afootnote{b c d e f}}
\edtext{\sameword[1]{B}}{\Afootnote{bb}}
\edtext{\sameword[1]{F}}{\Afootnote{ff}}
\pend
\endnumbering


\end{document}
```
instead of 
```latex
\documentclass{article}

\usepackage[series={A},nofamiliar,noeledsec,noledgroup,draft]{reledmac}

\begin{document}

\beginnumbering
\pstart
\edtext{\sameword{B}}{\Afootnote{del.}}
\edtext{\sameword{F}}{\Afootnote{del.}}
\edtext{\sameword[1]{B} C D E \sameword[1]{F}}{\lemma{\sameword{B}--\sameword{F}}\Afootnote{b c d e f}}
\edtext{\sameword{B}}{\Afootnote{bb}}
\edtext{\sameword{F}}{\Afootnote{ff}}
\pend
\endnumbering


\end{document}
```

**Comment by stenskjaer** [comment]
I actually do see the new comments in already closed issues, I just somehow managed to ignore it.

**Comment by stenskjaer** [comment]
I close this now as the problems indicated here are solved through issue #24.
